src/modules/invite.ts

```diff
@@ -229,13 +228,20 @@ const inviteCSVList = async (
     if (filteredInviteeList.length === 0) {
         throw new ClientError('No valid invitees');
     }
+    if (previewEmail)
+        filteredInviteeList.push({
+            email: previewEmail,
+            fName: 'fName',
+            lName: 'lName',
+        } as InviteeData);
     return inviteMany(
         filteredInviteeList,
         data.MoC, // Checked if undefined earlier
         data.topic,
         data.eventDateTime,
         data.constituentScope,
-        data.deliveryTime
+        data.deliveryTime,
+        data.townHallId
     );
```

Comment:
I didn't notice this last time, but couldn't we also just pass in the entire object to avoid having so many function arguments?

Reply:
Fixed